### ISSUE TITLE
Rule Proposal: vue/class-order

### ISSUE THREAD

**Issue by mchmurski-rms** [vuejs/eslint-plugin-vue#885]
**Please describe what the rule should do:**
enforce classnames alphabetical order

**What category should the rule belong to?**

- [ ] Enforces code style
- [ ] Warns about a potential error
- [x] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template><div class="b a"></div></template>
<template><div class="a c b"></div></template>
```

**Additional context**

When working with classes provided by design system, having a fixed order improves visibility.

**Comment by ota-meshi** [comment]
I close this issue because we merged #886.
